src/sentry/auth/staff.py

```diff
@@ -51,6 +52,16 @@ def is_active_staff(request: HttpRequest | Request) -> bool:
     return staff.is_active
 
 
+# TODO(schew2381): Delete this method after the option is removed
+def has_staff_option(user) -> bool:
+    """
+    Checks if the user can access the staff mode for _admin depending on an option.
+    """
+    if (email := getattr(user, "email", None)) is None:
+        return False
+    return email in options.get("staff.user-email-allowlist")
```

Comment:
how will we roll this out to everyone without putting everyone's email on the allowlist?

Reply:
I plan on creating a boolean option and then checking that first in the helper like:
```Python
def has_staff_option(...):
    if options.get("staff.ga-enabled"):
        return True
    ...
```